content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/README.md

```diff
@@ -34,6 +34,17 @@ The following configuration properties are used:
 2. `./embeddablesDisabled` - defines whether or not embeddables are disabled in the edit dialog.
 1. `./htmlDisabled` - defines whether or not free-form HTML input is disabled in the edit dialog.
 3. `./allowedEmbeddables` - defines the embeddables that are allowed to be selected by an author when embeddables are not disabled.
+4. `./youtubeMuteEnabled` - enables the ability of content authors to configure the [YouTube mute parameter][yt-parameters]
```

Comment:
I would prefer these are documented in `apps/core/wcm/components/embed/v1/embed/embeddable/youtube/README.md` as they are specific to this embeddable.